gslb/cache/controller_obj_cache.go

```diff
@@ -627,24 +629,70 @@ func ParsePoolAlgorithmSettingsFromPoolRaw(group map[string]interface{}) *gslbal
 		hashMaskI := int32(hashMaskF)
 		consistentHashMask = &hashMaskI
 	} else {
-		gslbutils.Logf("couldn't parse hash mask: %v", group["consistent_hash_mask"])
+		gslbutils.Debugf("couldn't parse hash mask: %v", group)
```

Comment:
should this be a warning log?

Reply:
`consistent_hash_mask` will be only present when the `algorithm` or `fallback_algorithm` is set as consistent hash.

Reply:
As discussed, I have added a check whether the `algorithm` or `fallback_algorithm` is consistent hash, then only access the `consistent_hash_mask`.